### DISCUSSION TITLE
Doris 查询结果乱序

### DISCUSSION THREAD

**Discussion by hxy317** [apache/doris#43420] [A - General / Q&A]
使用doris 2.1.5 查询doris表时候，使用group by 之后，如果不使用order by  结果集是无序的。执行多次，返回结果顺序不一致，
在doris 1.0.x 时候是有默认排序的，是否有相关的配置

**Comment by echo-hhj** [comment]
2.1.5的行为是正确的，排序对性能会有损耗的，如果想要排序需要加order by来进行排序。

**Comment by htyoung** [comment]
不加order by不保证多次执行返回数据一样的，这是没问题的，很数据数据库引擎都是这样
